lib/command/src/Obelisk/Command/Nix.hs

```diff
@@ -180,3 +186,16 @@ nixCmd cmdCfg = withSpinner' ("Running " <> cmd <> desc) (Just $ const $ "Built
       [ (" on " <>) <$> path
       , (\a -> " [" <> a <> "]") <$> (commonCfg ^. nixCmdConfig_target . target_attr)
       ]
+
+data NValue
+  = NValue_Text Text
+  | NValue_Path FilePath
+
+renderNValue :: NValue -> Text
+renderNValue = \case
+  NValue_Text x -> "\"" <> x <> "\""
+  NValue_Path p -> T.pack p
+
+renderAttrset :: Map Text NValue -> Text
+renderAttrset m = "{ " <> T.unwords (map (\(k,v) -> k <> " = " <> renderNValue v <> ";") $ M.toList m) <> " }"
+
```

Comment:
If we are pretty-printing nix, please us hnix.

Reply:
:+1: